pype/plugins/maya/publish/collect_review.py

```diff
@@ -69,7 +69,16 @@ def process(self, instance):
                     instance.data['remove'] = True
                 i += 1
         else:
-            instance.data['subset'] = task + 'Review'
+            if instance.data.get("legacy", True):
```

Comment:
This should rather be a class attribute, that way it is configurable per project and per studio. It will make the transition much easier. 

It's not something that the artist should have control over, but studio config should.

Reply:
Cool, that makes sense. Done.